store/sqlstore/tokens_store.go

```diff
@@ -74,11 +74,15 @@ func (s SqlTokenStore) Cleanup() {
 	}
 }
 
-func (s SqlTokenStore) RemoveAllTokensByType(tokenType string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		if _, err := s.GetMaster().Exec("DELETE FROM Tokens WHERE Type = :TokenType", map[string]interface{}{"TokenType": tokenType}); err != nil {
-			result.Err = model.NewAppError("SqlTokenStore.RemoveAllTokensByType", "store.sql_recover.remove_all_tokens_by_type.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
-	})
+func (s SqlTokenStore) RemoveAllTokensByType(tokenType string) *model.AppError {
+	if _, err := s.GetMaster().Exec(
```

Comment:
I would prefer to respect the previous indentation approach (Just one line for the return and for the Exec).

Reply:
Point taken, amended the commit